jenkins/check-for-build.jenkinsfile

```diff
@@ -11,11 +11,10 @@ pipeline {
     }
     triggers {
         parameterizedCron '''
-            H 1 * * * %INPUT_MANIFEST=1.3.5/opensearch-dashboards-1.3.5.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
-            H 1 * * * %INPUT_MANIFEST=2.2.0/opensearch-dashboards-2.2.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
+            H/60 * * * * %INPUT_MANIFEST=2.2.0/opensearch-2.2.0.yml;TEST_MANIFEST=2.2.0/opensearch-2.2.0-test.ymlTARGET_JOB_NAME=distribution-build-opensearch
+            H/60 * * * * %INPUT_MANIFEST=2.2.0/opensearch-dashboards-2.2.0.yml;TTEST_MANIFEST=2.2.0/opensearch-dashboards-2.2.0-test.yml;ARGET_JOB_NAME=distribution-build-opensearch-dashboards
```

Comment:
Missed `;` in first and `;T` in second

Reply:
Extra `T` in the spelling as well.

Reply:
I think I missed the copy location, should fix now.
Thanks.